pages/agent/v3/help/_start.md

```diff
@@ -38,9 +38,10 @@ The agent will run any jobs within a PTY (pseudo terminal) if available.
 <tr id="priority"><th><code>--priority value</code> <a class="Docs__attribute__link" href="#priority">#</a></th><td><p>The priority of the agent (higher priorities are assigned work first)<br /><strong>Environment variable</strong>: <code>$BUILDKITE_AGENT_PRIORITY</code></p></td></tr>
 <tr id="acquire-job"><th><code>--acquire-job value</code> <a class="Docs__attribute__link" href="#acquire-job">#</a></th><td><p>Start this agent and only run the specified job, disconnecting after it's finished<br /><strong>Environment variable</strong>: <code>$BUILDKITE_AGENT_ACQUIRE_JOB</code></p></td></tr>
-<tr id="disconnect-after-job"><th><code>--disconnect-after-job </code> <a class="Docs__attribute__link" href="#disconnect-after-job">#</a></th><td><p>Disconnect the agent after running a job<br /><strong>Environment variable</strong>: <code>$BUILDKITE_AGENT_DISCONNECT_AFTER_JOB</code></p></td></tr>
-<tr id="disconnect-after-idle-timeout"><th><code>--disconnect-after-idle-timeout value</code> <a class="Docs__attribute__link" href="#disconnect-after-idle-timeout">#</a></th><td><p>If no jobs have come in for the specified number of seconds, disconnect the agent (default: 0)<br /><strong>Environment variable</strong>: <code>$BUILDKITE_AGENT_DISCONNECT_AFTER_IDLE_TIMEOUT</code></p></td></tr>
+<tr id="disconnect-after-job"><th><code>--disconnect-after-job </code> <a class="Docs__attribute__link" href="#disconnect-after-job">#</a></th><td><p>Disconnect the agent after running exactly one job. When used in conjunction with the --spawn flag, each worker booted will run exactly one job<br /><strong>Environment variable</strong>: <code>$BUILDKITE_AGENT_DISCONNECT_AFTER_JOB</code></p></td></tr>
+<tr id="disconnect-after-idle-timeout"><th><code>--disconnect-after-idle-timeout value</code> <a class="Docs__attribute__link" href="#disconnect-after-idle-timeout">#</a></th><td><p>The maximum idle time in seconds to wait for a job before disconnecting. The default of 0 means no timeout (default: 0)<br /><strong>Environment variable</strong>: <code>$BUILDKITE_AGENT_DISCONNECT_AFTER_IDLE_TIMEOUT</code></p></td></tr>
 <tr id="cancel-grace-period"><th><code>--cancel-grace-period value</code> <a class="Docs__attribute__link" href="#cancel-grace-period">#</a></th><td><p>The number of seconds a canceled or timed out job is given to gracefully terminate and upload its artifacts (default: 10)<br /><strong>Environment variable</strong>: <code>$BUILDKITE_CANCEL_GRACE_PERIOD</code></p></td></tr>
+<tr id="enable-job-log-tmpfile"><th><code>--enable-job-log-tmpfile </code> <a class="Docs__attribute__link" href="#enable-job-log-tmpfile">#</a></th><td><p>BUILDKITE_JOB_LOG_TMPFILE  Store the job logs in a temporary file BUILDKITE_JOB_LOG_TMPFILE that is accessible during the job and removed at the end of the job<br /><strong>Environment variable</strong>: <code>$BUILDKITE_ENABLE_JOB_LOG_TMPFILE</code></p></td></tr>
```

Comment:
Is there an extra space in "BUILDKITE_JOB_LOG_TMPFILE  Store the job logs" (between BUILDKITE_JOB_LOG_TMPFILE and Store)?

Reply:
Yep, you're right! I'll fix this on the agent side and then update here (the docs here are generated from agent code)